packages/plugin-aws-lambda/src/index.js

```diff
@@ -0,0 +1,84 @@
+const bugsnagInFlight = require('@bugsnag/in-flight')
+
+const BugsnagPluginAwsLambda = {
+  name: 'awsLambda',
+
+  load (client) {
+    bugsnagInFlight.trackInFlight(client)
+
+    return {
+      createHandler ({ flushTimeoutMs = 2000 } = {}) {
+        return wrapHandler.bind(null, client, flushTimeoutMs)
+      }
+    }
+  }
+}
+
+function wrapHandler (client, flushTimeoutMs, handler) {
+  let _handler = handler
+
+  if (handler.length > 2) {
+    // This is a handler expecting a 'callback' argument, so we convert
+    // it to return a Promise so '_handler' always has the same API
+    _handler = promisifyHandler(handler)
+  }
+
+  return async function (event, context) {
+    client.addMetadata('AWS Lambda context', context)
+
+    try {
+      return await _handler(event, context)
+    } catch (err) {
+      if (client._config.autoDetectErrors && client._config.enabledErrorTypes.unhandledExceptions) {
+        const handledState = {
+          severity: 'error',
+          unhandled: true,
+          severityReason: { type: 'unhandledException' }
+        }
+
+        const event = client.Event.create(err, true, handledState, 1)
```

Comment:
`Event.create` has the signature `(maybeError, tolerateNonErrors, handledState, component, errorFramesToSkip = 0, logger)` – is `1` meant to be the value for `framesToSkip`?

Reply:
Yeah, I think this was a mistake [copied from the Vue plugin](https://github.com/bugsnag/bugsnag-js/blob/next/packages/plugin-vue/src/vue.js#L6) — I'll make a ticket to fix it there too